stograde/student/markdownify/add_remove_supporting.py

```diff
@@ -0,0 +1,34 @@
+import os
+
+
+def import_supporting(*, spec, spec_id, basedir):
+    cwd = os.getcwd()
+    inputs = spec.get('inputs', [])
+    supporting_dir = os.path.join(basedir, 'data', 'supporting')
+    written_files = []
+    # write the supporting files into the folder
+    for filename in inputs:
+        if isinstance(filename, list):
+            if len(filename) == 1:
+                in_name = filename[0]
+                out_name = filename[0]
+            else:
+                in_name = filename[0]
+                out_name = filename[1]
+        else:
+            in_name = filename
+            out_name = filename
+        with open(os.path.join(supporting_dir, spec_id, in_name), 'rb') as infile:
```

Comment:
Blank line before this, please, to separate the in_name/out_name discovery from the file actions